lib/cosmos/gui/dialogs/about_dialog.rb

```diff
@@ -37,28 +40,45 @@ class AboutDialog < Qt::Dialog
 
     @@pry_dialogs = []
 
-    def initialize (parent, about_string)
+    def initialize(parent, about_string)
       super(parent, Qt::WindowTitleHint | Qt::WindowSystemMenuHint)
       @saved_text = ''
       setWindowTitle('About')
 
-      # Get Word Icon
       filename = File.join(::Cosmos::USERPATH, 'config', 'data', 'cosmos_word.gif')
       filename = File.join(::Cosmos::PATH, 'data', 'cosmos_word.gif') unless File.exist?(filename)
       word_icon = Qt::Label.new
       word_icon.setPixmap(Qt::Pixmap.new(filename))
 
-      copyright = Qt::Label.new("Copyright 2014 - Ball Aerospace & Technologies Corp.")
+      copyright = Qt::Label.new("Copyright 2017 - Ball Aerospace & Technologies Corp.")
```

Comment:
If you change the code, you can up the copyright date.   Modified files are copyrighted from the date of modification.